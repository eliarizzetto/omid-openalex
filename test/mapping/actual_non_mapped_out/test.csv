"omid","type"
"omid:br/060924","report"
"omid:br/06076","book series"
"omid:br/06085","series"
"omid:br/060183",""
"omid:br/060184","dataset"
"omid:br/0601599","book chapter"
"omid:br/0601657","journal article"
"omid:br/06201135579","journal"
"omid:br/06201147464","journal"
"omid:br/06201235690","journal"
"omid:br/06201270416","journal"
"omid:br/06203990365","computer program"
"omid:br/062303878985","data management plan"
