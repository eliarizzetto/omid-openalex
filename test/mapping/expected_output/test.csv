"omid","openalex_id","type"
"omid:br/06041","W2006761296","journal article"
"omid:br/060181","W4299951447","book"
"omid:br/0601649","W4253101493","journal article"
"omid:br/06103119650","W2104018839","journal article"
"omid:br/06103133864","W4240841444","proceedings"
"omid:br/06201860301","W3044146539","web content"
"omid:br/06201860238","W3105616540","proceedings article"
"omid:br/06201860187","W3045555670","peer_review"
"omid:br/06201862562","W3164117194","book part"
"omid:br/06201862917","W3108429399","dissertation"
"omid:br/06201864757","W2749887630","reference entry"
"omid:br/06202696306","W4234930467","book set"
"omid:br/06202902524","W4253032323","book section"
